primitives/millau/src/lib.rs

```diff
@@ -155,6 +160,44 @@ impl sp_runtime::traits::Convert<sp_core::H256, AccountId> for AccountIdConverte
 		hash.to_fixed_bytes().into()
 	}
 }
+
+/// Get a struct which defines the weight limits and values used during extrinsic execution.
+pub fn runtime_block_weights() -> frame_system::limits::BlockWeights {
```

Comment:
I moved the builder call (and the length builder below) here because I didn't want other parts of the codebase to be pulling in the runtime code as a dependency, but this isn't super nice. Two questions here:

1. Is it okay for me to be pulling stuff in from `frame_system` here?
2. Is there any way for me to get a shared instance of this that I can use in the runtime but also in the relay and message lane crates?

Reply:
@tomusdrw Hey can you take a look at this? I feel like you have opinions about this 😛

Reply:
This looks good.
1. Yes, it actually has less dependencies than `frame-support` anyway.
2. Long term relay should rely on the metadata instead of the primitives directly.